src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description)
+    {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public Task()
+    {
+        this.description = "empty";
+        this.isDone = false;
+    }
+
+    public void markAsDone()
```

Comment:
Consider writing a `JavaDoc` comment for public methods, as it is meant to used by others. This seems to be a violation of this [coding standard](https://se-education.org/guides/conventions/java/index.html#comments), more specifically it is advisable to write descriptive header comments for all public classes/methods.